__tests__/app.test.js

```diff
@@ -42,6 +42,70 @@ describe('app', () => {
             })
         })
         describe('/reviews', () => {
+            describe('GET: /api/reviews', () => {
+                test('status 200: responds with array of reviews that have comment_count key', () => {
+                    return request(app)
+                    .get('/api/reviews')
+                    .expect(200)
+                    .then(({ body }) => {
+                        const { reviews } = body;
+                        expect(reviews).toBeInstanceOf(Array);
+                        expect(reviews).toHaveLength(13);
+                        reviews.forEach(review => {
+                            expect(review).toEqual(
+                                expect.objectContaining({
+                                    owner: expect.any(String),
+                                    title: expect.any(String),
+                                    review_id: expect.any(Number),
+                                    category: expect.any(String),
+                                    review_img_url: expect.any(String),
+                                    created_at: expect.any(String),
+                                    votes: expect.any(Number),
+                                    designer: expect.any(String),
+                                    comment_count: expect.any(Number)
+                                })
+                            )
+                        })
+                    })
+                })
+                test('status 200: reviews are sorted by date in descending order by default', () => {
+                    return request(app)
+                    .get('/api/reviews')
+                    .expect(200)
+                    .then(({ body }) => {
+                        const { reviews } = body;
+                        expect(reviews).toBeSortedBy('created_at', { descending: true })
+                    })
+                })
+                test('status 200: allows reviews to be sorted by category', () => {
+                    return request(app)
+                    .get("/api/reviews?category=social deduction")
+                    .expect(200)
+                    .then(({ body }) => {
+                        const { reviews } = body;
+                        expect(reviews).toHaveLength(11);
+                        reviews.forEach(review => {
+                            expect(review.category).toBe('social deduction')
+                        })
+                    })
+                })
+                test('status 404: responds with error if passed an invalid category query', () => {
+                    return request(app)
+                    .get('/api/reviews?category=bananas')
+                    .expect(404)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('Not found')
+                    })
+                })
+                test('status 404: responds with error if valid category but no results', () => {
```

Comment:
You should change this assertion, if the category exists then I would expect a 200 status and an empty array to be returned